misc/config_tools/xforms/lib.xsl

```diff
@@ -420,7 +420,7 @@
                 <node><xsl:value-of select="$res_clos_max"/></node>
               </xsl:when>
               <xsl:otherwise>
-                <node><xsl:value-of select="$res_clos_max div (1 + $cdp_enabled)"/></node>
+                <node><xsl:value-of select="floor($res_clos_max div (1 + $cdp_enabled))"/></node>
```

Comment:
The problem here is not the lack of a `floor`, but `max` is misused as `count`.

In case we want to reduce the number of CLOS available by half, the correct formula should be:

`($res_clos_max + 1) div (1 + $cdp_enabled) - 1`

Reply:
Not sure if we will get new board xmls for ehl, tgl...etc. Seems they are either manual edited or really get even numbers in <CLOS_INFO>. I leave "floor" here for back compatibility.